test/browser_tests/page_objects/page_blog-single.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+function BlogSingle() {
+
+  this.get = function() {
+    browser.get( '/blog/lessons-weve-learned/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.postTitle = element( by.css( 'header .post_heading' ) );
+
+  this.postByLine = element( by.css( 'header .post_byline' ) );
+
+  this.postBody = element( by.css( '.post .post_body' ) );
+
+  this.share = element( by.css( 'header .share' ) );
+
+  this.breadcrumbs = element( by.css( '.breadcrumbs' ) );
+
+  this.tags = element( by.css( '.tags' ) );
+
+  this.contentSidebar = element( by.css( '.content_sidebar' ) );
+
+  this.relatedPosts =
+  this.contentSidebar.element( by.css( '.related-posts' ) );
+
+  this.relatedPostsTitle =
+  this.relatedPosts.element( by.css( '.header-slug_inner' ) );
+
+  this.stayInformedSection = this.contentSidebar
+  .element( by.css( '[data-qa-hook="stay-informed-section"]' ) );
+
+  this.stayInformedSectionTitle =
+  this.stayInformedSection.element( by.css( '.header-slug_inner' ) );
+
+  this.emailSubscribeForm =
+  this.stayInformedSection.element( by.css( '#email-subscribe-form' ) );
+
+  this.emailSubscribeForm =
+  this.stayInformedSection.element( by.css( '#email-subscribe-form' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/f18474a240fd0f56bdd225730736293f2f28c11c